students/skowronski_rafal/lesson_01_basics/lesson_02/problem_03.py

```diff
@@ -0,0 +1,16 @@
+# Solves problem 03 - Sum of digits
+import math
+
+
+def print_sum_of_digits():
+    number = abs(int(input('Enter integer number: ')))
+
+    sum = 0
```

Comment:
'sum' is build in function. You redefined it.
>>> sum([1,2,3])
6
>>> sum = 10
>>> sum([1,2,3])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'int' object is not callable

Be careful using built in as a variables